src/pages/partners/payment-agent/_faq-data.tsx

```diff
@@ -8,50 +8,20 @@ const General = () => {
     const { is_p2p_allowed_country } = React.useContext(DerivStore)
     return (
         <>
-            <Header
-                as="p"
-                type="paragraph-1"
-                mt="16px"
-                tabletL={{
-                    type: 'paragraph-2',
-                }}
-            >
+            <Header as="p" type="paragraph-1" mt="16px">
```

Comment:
Don't we need the tabletL prop here? Maybe the `type: 'paragraph-2' ` can be given a different type to fix the error

Reply:
Hi @konstantinosG-deriv , as per the design the size is 14px. So no need to add this prop.